trace-resources/README.md

```diff
@@ -0,0 +1,3 @@
+trace-resources
+
+Query resource statistics for various platforms for tracing.
```

Comment:
Please put the devs/team responsible for this library here

Reply:
Okay